operator/provider_aws_flags.go

```diff
@@ -45,6 +45,9 @@ func init() {
 
 	flags.Bool(operatorOption.UpdateEC2AdapterLimitViaAPI, false, "Use the EC2 API to update the instance type to adapter limits")
 	option.BindEnv(operatorOption.UpdateEC2AdapterLimitViaAPI)
+	flags.MarkDeprecated(operatorOption.UpdateEC2AdapterLimitViaAPI, "This option has been deprecated and will be removed in v1.10")
```

Comment:
It's best to point to the new flag when warning about a deprecated flag.

Reply:
Sure will add a reference to the new flag.